### PR TITLE
Improved SEO and performance for lighthouse

### PR DESCRIPTION
**Pull request by eamon0989**
## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
|     | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

### CODE REVIEW

**Comment by mrchrmn** [comment]
It's weird that the changes in manifest.json look like there was no content before.

File: src/components/home/Hero.tsx [hunk 1]
```diff
@@ -7,11 +7,11 @@ export default function Benefits() {
...
-          <div className="items-center h-full flex flex-col md:flex-row ">
+          <div className="items-center h-auto flex flex-col md:flex-row ">
             {/* Section header */}
             <div className=" flex-1 flex justify-between h-max flex-col gap-6">
-              <h1 className={`${window.innerHeight < 800 ? 'text-4xl' : 'text-5xl'} text-center md:text-left xl:text-6xl lg:text-5xl md:text-4xl sm:text-6xl font-extrabold leading-tighter tracking-normal`} data-aos="zoom-y-out">Alexandria:<br />Learn languages by<br />
-                <span className="bg-clip-text block text-transparent bg-gradient-to-r from-[#167cbd] to-[#6a007b]">reading what you enjoy</span>
+              <h1 className={`${window.innerHeight < 800 ? 'text-4xl' : 'text-5xl'} text-center  overflow-y-clip md:text-left xl:text-6xl lg:text-5xl md:text-4xl sm:text-6xl font-extrabold leading-tighter tracking-normal`} data-aos="zoom-y-out">Alexandria:<br />Learn languages by<br />
+                <span className="bg-clip-text block pb-3 mb-[-0.75rem] text-transparent bg-gradient-to-r from-[#167cbd] to-[#6a007b]">reading what you enjoy</span>
```
Comment (mrchrmn) [inline src/components/home/Hero.tsx:14]: Adding `mb-[-0.75rem]` (which wasn't there a moment ago when I tried) reverts the effect and the bottom of the y is cut off again.

**Review by mrchrmn — CHANGES_REQUESTED** [review]

File: src/components/home/Hero.tsx (hunk 1 above)
Comment (eamon0989) [reply src/components/home/Hero.tsx:14]: I think your computer is broken 😁 It works perfectly here. I can remove it.

Comment (mrchrmn) [reply src/components/home/Hero.tsx:14]: I have this on Windows and Linux with Edge, Firefox and Brave. Might be just me, or you're the lucky one.

**Review by mrchrmn — APPROVED** [review]

**Pull request merged**